plugins/graphql-voyager/package.json

```diff
@@ -35,7 +35,7 @@
     "@material-ui/core": "^4.12.2",
     "@material-ui/icons": "^4.9.1",
     "@material-ui/lab": "4.0.0-alpha.61",
-    "graphql-voyager": "^1.0.0-rc.31",
+    "graphql-voyager": "^2.0.0",
```

Comment:
FYI:
graphql-voyager v2.0.0 upgraded from _(previously at RC31)_ material-ui v4 to v5

```
    "@mui/icons-material": "5.11.0",
    "@mui/lab": "5.0.0-alpha.114",
    "@mui/material": "5.11.2",
```

PS: This PR could resolve #19327 and #19326.

The RC31 was dependent on `"@material-ui/core": ^3.9.3` -> `recompose: 0.28.0 - 0.30.0` -> `fbjs: ^0.8.1` -> `isomorphic-fetch: ^2.1.1` -> `node-fetch: ^1.0.1`